snapshots/td_heat_conduction/readme.md

```diff
@@ -0,0 +1 @@
+This directory will hold the template case for the time dependent heat conduction. 
```

Comment:
use present tense, as if it were already there, otherwise you will have the change this file again soon